spec/features/pool_tubes_at_end_of_pipeline_spec.rb

```diff
@@ -12,11 +12,15 @@
   let(:tube_uuid) { SecureRandom.uuid }
   let(:sibling_uuid) { 'sibling-tube-0' }
   let(:child_purpose_uuid) { 'child-purpose-0' }
-  let(:example_v2_tube) do
-    create(:v2_tube, uuid: tube_uuid, state: 'passed', barcode_number: 1, purpose_name: 'Example Purpose')
-  end
-  let(:example_tube) do
-    json(:tube_with_siblings, uuid: tube_uuid, siblings_count: 1, state: 'passed', barcode_number: 1)
+  let(:tube) do
+    create(
+      :v2_tube,
+      barcode_number: 1,
+      purpose_name: 'Example Purpose',
+      siblings_count: 1,
```

Comment:
Maybe this is slightly confusing, (only if my above comment is correct), that is you put `siblings_count: 1`, it is actually creating 2, one by default and one extra?

Reply:
I think it's important to maintain the same functionality as the v1 factories, even if we don't agree with it.  I don't necessarily understand why it was written that way, but we're trying to prove the new code changes do not have any different behaviours under v2, so we have to swallow weird mocking choices.